### PR TITLE
Add skript to plot skymap

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

**Review by jebuss — APPROVED** [review]
Thanks for cleaning up. Looks good, nothing to complain.

File: matplotlibrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+backend: Qt5Agg
```
Comment (kbruegge) [inline matplotlibrc]: why do we need a matplotlibrc in here? will it get used when I call the executables from somewhere else?

Comment (kbruegge) [reply matplotlibrc]: Also. why qt5agg? and serif?

Comment (maxnoe) [reply matplotlibrc]: I accidentally added the file

**Review by kbruegge — APPROVED** [review]

**Pull request merged**